src/screens/ProfileScreen/index.js

```diff
@@ -103,7 +103,7 @@ const Header = (props) => {
           <ProfilePicture
             onImageContainerClick={changeImage}
             profilePicPath={dataMain.profile_pic_path}
-            karmaScore={Math.floor(dataMain.karma_score)}
+            karmaScore={dataMain.karma_score}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from rounding down the `karmaScore` to passing it directly might cause issues if the `ProfilePicture` component or any of its child components expect an integer value for `karmaScore`. If fractional values can cause problems, consider keeping the `Math.floor()` function.
```diff
- karmaScore={Math.floor(dataMain.karma_score)}
+ karmaScore={dataMain.karma_score}
```
If the `ProfilePicture` component and its children can handle fractional `karmaScore` values without any issues, then this change is fine.


<!-- This is an auto-generated comment by OSS CodeRabbit -->